packages/shell-api/src/field-level-encryption.ts

```diff
@@ -47,12 +50,16 @@ export interface ClientSideFieldLevelEncryptionOptions {
 @classPlatforms([ ReplPlatform.CLI ] )
 export class ClientEncryption extends ShellApiClass {
   public _mongo: Mongo;
-  public _libmongocrypt: any;
+  public _libmongocrypt: MongoCryptClientEncryption;
 
-  constructor(mongo: any) {
+  constructor(mongo: Mongo) {
     super();
     this._mongo = mongo;
-    this._libmongocrypt = new mongo._serviceProvider.fle.ClientEncryption(
+    const fle = mongo._serviceProvider.fle;
+    if (!fle) {
+      throw new MongoshRuntimeError('FLE API is not available');
```

Comment:
I think this is where https://jira.mongodb.org/browse/MONGOSH-514 comes into play :)

Reply:
Yep :) Just wanted to get rid of the TypeScript error with undefined things... :D

Reply:
Yeah this is literally what I was going to put in, so once this is merged I'll just close the ticket. Nice!